use-cases/gst_capi/add_camera-simulator.yml

```diff
@@ -0,0 +1,81 @@
+#
```

Comment:
is it possible to reuse the file from the grpc_python folder? or are there settings that are different? seems redundant to have the file nearly duplicated. like in the grpc_python one we could take out the OVMS pieces and put them into a different file.

Reply:
Agree, we can reuse the camera simulator docker compose for all other use cases

Reply:
Hi @ejlee3  and @antoniomtz- 
So this is separation principle:  that's different use cases which i consider should be separate. i can see a little bit of duplication of camera simulator in docker-compose file but not too much. Thus, I would rather be willing to pay that price to keep the isolation of different use cases to be more self-contained.

Reply:
I see - so I guess the question I think we need to answer is how do we want things to be separate? do we want every use case to have its own compose file or would we want to combine things so that we have building blocks we can use for different use cases? I think the latter is more scalable and could be more developer friendly, but it really depends on what we see as the future of the project and how we decide to separate things out.

Reply:
Hi @ejlee3    yes, right now I'm focusing on the docker-compose for each use case as a separation boundary. I see your good point too about the building block. The bright side is that the code / docker image for camera-simulator is actually a building block already and it is pulling from sub-module performance-tools repo. Hence to a certain degree we have that goal achieved already.

Reply:
one more point to add on: you can think of "docker-compose" as a use case packaging tool if you will; so each use case could get different docker-compose files

Reply:
Resolved as per yesterday's discussion meeting and we could do the refactor of sharing docker-compose snippets across all use cases in future if found that is necessary and fit.